src/main/java/com/wellsky/workflow/WorkflowApplication.java

```diff
@@ -6,7 +6,14 @@
 @SpringBootApplication
 public class WorkflowApplication {
 
+    private String Password = "not real still bad";
+
     public static void main(String[] args) {
         SpringApplication.run(WorkflowApplication.class, args);
+        fakeMethod(null);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') "NullPointerException" will be thrown when invoking method "fakeMethod()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://hhh-aks.eastus.cloudapp.azure.com/coding_rules#rule_key=squid%3AS2259)